sql_app/schemas.py

```diff
@@ -0,0 +1,29 @@
+"""Schemas for common attributes while creating or reading data."""
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class IdeaBase(BaseModel):
+    title: str
+    description: Optional[str] = None
+    date: datetime
+    # TODO: image: file --> how do I do this?
```

Comment:
images are just bytes so something like: `image: bytes` should be able to store the pic.